sample/src/main/java/com/nononsenseapps/filepicker/sample/MultimediaPickerFragment.java

```diff
@@ -78,6 +93,10 @@ public void onBindViewHolder(DirViewHolder vh, int position, File file) {
         if (isImage(file)) {
             vh.icon.setVisibility(View.VISIBLE);
             Glide.with(this).load(file).centerCrop().into((ImageView) vh.icon);
+        } else if (isVideo(file)) {
+            vh.icon.setVisibility(View.VISIBLE);
```

Comment:
Not much point in having to blocks with the exact same code. Would make sense to unify them I think, something like

```java
if (isMultimedia(file)) {
//...
}
```

or

```java
if (isPreviewable(file)) {
//...
}
```